modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts

```diff
@@ -439,13 +439,12 @@ interface GetCurrentContentHandler {
  * @returns {object|null} The current content as JSON.
  */
 export function getContent<
-    Data = Record<string, unknown>,
-    Type extends string = string,
+    DataAndType extends DataAndTypeTuple = readonly [data: Record<string, unknown>, type: string],
```

Comment:
Same question here.

Reply:
So, while `getContent()` will only return one answer, we don't necessarily know the content type in advance.

Imagine a part that can be used on different content types. We don't know which exact type `getContent` will return, but we can use if-statements to handle each possibility (or return e.g. a 404 if it was none of them).